README.md

```diff
@@ -24,27 +24,6 @@ dependencies {
 }
 ```
 
-#### Android SDK Versions Troubleshooting
```

Comment:
No longer valid

Reply:
Are there any other README updates related to this change that we can add now? Rather than doing a larger READEM update later, if we can incrementally update it with the work as we go, will make it easier on ourselves.

Reply:
There will be more changes (e.g. guidance around OIDC flag, or changes in the Credentials Manager) but I don't think these can be considered part of the AndroidX migration. I'd leave those for separate PRs.